README.md

```diff
@@ -314,73 +314,38 @@ See [third_party_integration](./third_party_integration) for examples of code mo
 This repository is tested on Python* 3.8.10, PyTorch* 1.12.1 (NVidia CUDA\* Toolkit 11.6) and TensorFlow* 2.8.4 (NVidia CUDA\* Toolkit 11.2).
 
 ## Installation
-We suggest to install or use the package in the [Python virtual environment](https://docs.python.org/3/tutorial/venv.html).
-
-If you want to optimize a model from PyTorch, install PyTorch by following [PyTorch installation guide](https://pytorch.org/get-started/locally/#start-locally). 
-If you want to optimize a model from TensorFlow, install TensorFlow by following [TensorFlow installation guide](https://www.tensorflow.org/install/).
-
-#### As a package built from a checked-out repository:
-
-Install the package and its dependencies by running the following in the repository root directory:
-```
-pip install .
-```
-
-Note that if you install NNCF in this manner, the backend frameworks supported by NNCF will not be explicitly installed. NNCF will try to work with whatever backend versions you have installed in your Python environment.
-
-If you want to install both NNCF and the supported PyTorch version in one line, you can do this by running:
-```
-pip install .[torch]
-```
-For installation of NNCF along with TensorFlow, run:
-```
-pip install .[tf]
-```
-For installation of NNCF for ONNX, run:
-```
-pip install .[onnx]
-```
-(Preview) For installation of NNCF for OpenVINO, run:
-```
-pip install .[openvino]
-```
-
-
-_NB_: For launching example scripts in this repository, we recommend setting the `PYTHONPATH` variable to the root of the checked-out repository once the installation is completed.
+For detailed installation instructions please refer to the [Installation](./docs/Installation.md) page.
 
 #### As a PyPI package:
 
 NNCF can be installed as a regular PyPI package via pip:
 ```
 pip install nncf
 ```
-Use the same `pip install` syntax as above to install NNCF along with the backend package versions in one go, i.e. for NNCF with PyTorch, run:
+If you want to install both NNCF and the supported PyTorch version in one line, you can do this by simply running:
```

Comment:
I think we can remove the information about extra parameters from the readme file and add the information that the user has to install the framework themselves.

cc' @AlexKoff88 @MaximProshin @vshampor

Reply:
I think it's useful to keep it since we have it and it's convenient and users actually **don't** have to install framework themselves

Reply:
I would keep it as it's one of the installation paths of nncf